### ISSUE TITLE
backup anaconda downloads

### ISSUE THREAD

**Issue by sdtaylor** [Wolflab/2017-09-29-DC_USU#3]
The download links for anaconda python were not working for me on USU guest wifi access, so I've set these up just in case

[Windows 64bit](https://drive.google.com/open?id=0ByNGBGdHMdc2SjFINzdVUjJmYkk)
[Windows 32bit](https://drive.google.com/file/d/0ByNGBGdHMdc2Y2xGUUxsM050Y2M/view?usp=sharing)

[Mac](https://drive.google.com/open?id=0ByNGBGdHMdc2RFZleldZbll1dWc)

[Linux 32bit](https://drive.google.com/open?id=0ByNGBGdHMdc2NEF4TFY2Mmd2cUU)
[Linux 64bit](https://drive.google.com/open?id=0ByNGBGdHMdc2bTlhUTNpSktPWFE)

**Comment by Wolflab** [comment]
Did that email go to all participants, or should I send that out? I did not get any complaints yet but we’ll find out in three hours

Paul

**Comment by sdtaylor** [comment]
No, this only went to us. I just wanted to have these links on hand in case
anybody needs them.

**Comment by Wolflab** [comment]
Ok- meanwhile, looking over the first part of the python lesson and notice that the lesson plan shows all the python prompts, as if being run from command line. I don’t see prompts (>>> or …) in Jupyter notebook. Is there is  way to switch the on in notebook I will do that. Otherwise, I was thinking of starting in command line, then switch over to the notebook because it is easier visually.  Suggestions? Only ask because I know we have many python beginners and want to avoid the frequent confusion of prompts.

Thanks

Paul

**Comment by sdtaylor** [comment]
I'd say start in jupyter from the get go. it is a little cumbersome for typing one line examples. but for any sort of multiline if statement or for loop it's much better for the student. It also lets them see all their work more easily than the command line.

**Comment by Wolflab** [comment]
OK
